### PR TITLE
Fixes an inaccuracy with computer typing handling.

### PR DESCRIPTION
**Pull request by Iamgoofball**

### CODE REVIEW

**Comment by CrackParrot** [comment]
Make it so that i can play the game in one hand, and masturbate in the other.

**Comment by SgtxRootbeer** [comment]
@Iamgoofball add wearing of gloves with only one hand, and then re-design this so it only works if both hands have gloves, ok?

File: code/game/machinery/computer/_computer.dm [hunk 1]
```diff
@@ -69,6 +69,14 @@
 	update_icon()
 	return
 
+/obj/machinery/computer/attack_hand(mob/user)
+	..()
+	if(ishuman(user))
+		var/mob/living/carbon/human/H = user
+		if(H.gloves && istype(H.gloves, /obj/item/clothing/gloves/boxing))
+			to_chat(H, "<span class = 'warning'>You can't type with boxing gloves on. Are you stupid?"</span>")
+			return FALSE
```
Comment (ShizCalev) [inline code/game/machinery/computer/_computer.dm]: just return

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Comment by Cyberboss** [comment]
Real solution is to make boxing gloves required to be in your hands. This would be better than adding snowflake checks everywhere

**Comment by KorPhaeron** [comment]
snowflake

Maybe in the isadvancedtooluser check or something

**Pull request closed without merging**